### PR TITLE
Adds a white eyepatch

### PR DESCRIPTION
**Pull request by MOONPRINCESS17**
# About the pull request

Adds a white counterpart to the already existing eyepatch, which is black. This one appears exclusively in player loadouts and does not replace the classic eyepatch anywhere else.

# Explain why it's good for the game

For those of discerning fashion taste. That, or any possible... gimmicks such fashion might enable.

# Testing Photographs and Procedure
image
# Changelog
:cl:
add: Added a white eyepatch to player loadouts.
imageadd: Added sprites for a white eyepatch.
/:cl:

### CODE REVIEW

File: code/modules/clothing/glasses/glasses.dm [hunk 1]
```diff
@@ -194,6 +194,15 @@
 	flags_armor_protection = 0
 	flags_equip_slot = SLOT_EYES|SLOT_FACE
 
+/obj/item/clothing/glasses/eyepatch_white
```
Comment (Zonespace27) [inline code/modules/clothing/glasses/glasses.dm:197]: make this a subtype of /obj/item/clothing/glasses/eyepatch

**Review by sleepynecrons — CHANGES_REQUESTED** [review]
the whites are ridiculously bright. use colors from something like PMC gear if you're going to make white clothing
also is this not just you trying to add anime larp

**Comment by MOONPRINCESS17** [comment]
The item is now a subtype of the normal black eyepatch and the sprite is not as luridly bright.

File: code/modules/clothing/glasses/glasses.dm (hunk 1 above)
Comment (MOONPRINCESS17) [reply code/modules/clothing/glasses/glasses.dm]: Done

**Comment by sleepynecrons** [comment]
after speaking with other maintainers i will be closing the PR. a white eyepatch isn't something we want marines wearing as it's visually inconsistent with the faction. white is something we should keep reserved to PMCs/W-Y specific groups. the item is also just a reference to https://myanimelist.net/character/65865/Rikka_Takanashi

**Pull request closed without merging**